# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1320437489 25200
#      Fri Nov 04 13:11:29 2011 -0700
# Node ID f7bcac894ec6d817a78e5995e5991109e193a692
# Parent  9a8763bcf5758edaa58c53c02f72a986e33cf7f7
mexext.m: Add functional test.  Add seealso link to docstring.

* mexext.m: Add functional test.  Add seealso link to docstring.

diff --git a/scripts/miscellaneous/mexext.m b/scripts/miscellaneous/mexext.m
--- a/scripts/miscellaneous/mexext.m
+++ b/scripts/miscellaneous/mexext.m
@@ -14,13 +14,16 @@
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} mexext ()
 ## Return the filename extension used for MEX files.
+## @seealso{mex}
 ## @end deftypefn
 
 function retval = mexext ()
   retval = "mex";
 endfunction
+
+%!assert (mexext (), "mex")
